src/components/Filter/FilterSelectAddOption.vue

```diff
@@ -111,7 +111,7 @@ export default {
       url.upsertFilterOption(this.entityType, this.filterKey, value)
 
     },
-    getSuggestions: _.debounce(async function () {
+    getSuggestions: debounce(async function () {
       this.isLoading = true
```

Comment:
I will still attempt to remove this. Ideally, I don't want any artificial delays.
The goal is to have the Veu handle the "when" problem directly. 
However, in cases where it's unavoidable, we could defer execution to the end of the call stack 
using requestAnimationFrame or setTimeout, thereby avoiding the need for a lodash import.